### ISSUE TITLE
Feature request: catalogue mode

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/vstore#14]
I am aware it's too soon but if you think at this from the beginning, it could be easier in later. 

Catalogue mode means that no Add to cart button is displayed, not checkout, no cart. Admin area is the same, only frontend is different.

It should be work similar way as user/registration system, you set signup off and no button is displayed. 

Just idea.

**Comment by CaMer0n** [comment]
@Jimmi08 Would this be for the entire store, or just specific categories?

**Comment by Jimmi08** [comment]
@CaMer0n Cataloque mode is usually for all store. It's more about 2 in 1 solution (store and cataloque). 
But you are right, I saw this be set for products too (not sure if it was for category or for product)

More explanation: we have very strict laws for eshops in my country. Sometimes is easier to have just catalogue and a form to send order. If website has cart (or Add to cart button) , it is eshop and law is applicable, if not, it's cataloque and you can avoid eshop law. 
And I remember how hard was to find any catalogue solution (for example for collectors.)

**Comment by kreossino** [comment]
up this is very useful

**Comment by Moc** [comment]
@kreossino Bumping an issue will not really make a difference. The best you can do is show your support by adding a emoji (thumbs up - :+1: ) to the original issue report. You can do this by clicking on the top right of the post. Thanks :)

**Comment by CaMer0n** [comment]
@Jimmi08 If you wish to send a pull-request with the new preference and the code in the shortcodes to disable them.. I'll be happy to merge it.

**Comment by Jimmi08** [comment]
Are you sure? Because I am not if I am able to do it.  But I can try or better I would like to try :)

**Comment by kreossino** [comment]
I also asked my developer friend if can help us


Il 30/08/16 22:45, Jimako ha scritto:
>
> Are you sure? Because I am not if I am able to do it. But I can try or 
> better I would like to try :)
>

**Comment by Moc** [comment]
I should be able to find some time to add this in. No promises :)

**Comment by CaMer0n** [comment]
@Jimmi08 Have you considered added 'display:none' to the button elements to do what you requested?

**Comment by Jimmi08** [comment]
@CaMer0n Yes, but it's not solution, links would still work and somebody could display it.  I am waiting for boys to do something with this. This is not urgent for me, don't waste your time for it. Thanks

**Comment by Jimmi08** [comment]
@CaMer0n 
Could you look at this?
https://github.com/Jimmi08/vstore/commit/83b62684e9115cdf6244349a9ecd77e441ffccea
Catalog mode - Add to cart is removed (no button),  eshop links (pages) are removed (not available)
Equiry mode - Replaced Add to cart to Enquiry form (so button is stilll there, but with different text and action - popup window).

On second thoughts this is maybe better to do with separate plugin (vstore-cat or something).  I just don't know how with other plugin override vstore shortcodes.

**Comment by CaMer0n** [comment]
@Jimmi08 It looks good! I believe for "Inquiry Mode" it could be done with a custom template, replacing the "add to cart" with a URL or just plain HTML. In this case, I believe you would just need "Purchase Mode" and "Catalog Mode" as the 2 radio options, and being just one pref, it may not require its own tab.

**Comment by Jimmi08** [comment]
I agree. I just didn't know where to start. This plugin has label simple so your way is better. But I am now able to extended it with other plugin. I just created shortcodes class not extended from e_shortcode but from vstore shortcodes class and it looks that everything works. I normally avoid things I don't understand (what  I am not able to read from code) but this is amazing. And with e_admin I could test what I need without messing with your code. Just to find more time. Thanks for everything.
